web-app/js/portal/cart/WpsDownloadHandler.js

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2014 IMOS
+ *
+ * The AODN/IMOS Portal is distributed under the terms of the GNU General Public License
+ *
+ */
+
+Ext.namespace('Portal.cart');
+
+Portal.cart.WpsDownloadHandler = Ext.extend(Portal.cart.DownloadHandler, {
```

Comment:
@jkburges Yes now that the smoke has cleared, I agree with extracting the common code but i couldnt finish all tests in time. Feel free to take a look at https://github.com/aodn/aodn-portal/issues/1997 and make any changes to get it in if you wish